src/Menu.cpp

```diff
@@ -94,7 +94,7 @@ static MenuDef menuDefFile[] = {
     { _TRN("&Open...\tCtrl+O"),             CmdOpen,                   MF_REQ_DISK_ACCESS },
     { "Open Folder",                        CmdOpenFolder,             MF_REQ_DISK_ACCESS | MF_RAMICRO_ONLY },
     { _TRN("&Close\tCtrl+W"),               CmdClose,                  MF_REQ_DISK_ACCESS },

```

Comment:
@kjk 
NOT REQUIRED since it would need an extra translation changed entry ?
BUT existing &Close could be expanded to &Close Tab
Thus showing singularity distinct from the new multiple

Reply:
I'm sorry I'm unaware of how this translation feature works so any suggestions are appreciated, should I then change the originally written '&Close' to 'Close &Tab' as suggested?

Reply:
Also the ALT F T shortcut is yet to be written, does it have to be written in accelerators.cpp?

Reply:
@itsrdb
kjk will usually deal with translation as a separate requirement thus am NOT insisting existing is changed as there will be wider impact on existing translations, especially since it is used in a wider context.
the alt ft is automatic by adding the &Tab

Reply:
P.S If all is well you may wish to follow up after with website additions such as 
1 or 2 "Manual" pages
but wait till @kjk review is completed

Reply:
Sure @GitHubRulesOK